compiler/backend/presLangScript.sml

```diff
@@ -142,27 +146,26 @@ val con_to_pres_pat_def = tDefine"con_to_pres_pat"`
     case p of
        | conLang$Pvar varN => presLang$Pvar varN
        | Plit lit => Plit lit
-       | Pcon opt ps => ConPcon opt (MAP con_to_pres_pat ps)
+       | Pcon opt ps => Pcon (Con_con opt) (MAP con_to_pres_pat ps)
        | Pref pat => Pref (con_to_pres_pat pat)`
     cheat;
 
 val con_to_pres_exp_def = tDefine"con_to_pres_exp"`
-  (con_to_pres_exp (conLang$Raise t e) = Raise t (con_to_pres_exp e))
+  (con_to_pres_exp (conLang$Raise t e) = presLang$Raise t (con_to_pres_exp e))
```

Comment:
Actually, this might not be necessary. I will try to remove it and see if everything still works. The name of the function should be self explanatory.